panther_controller/launch/controller.launch.py

```diff
@@ -0,0 +1,177 @@
+            ),
+            " use_sim:=",
+            use_sim,
+            " simulation_engine:=",
+            simulation_engine,
+            " wheel_config_file:=",
+            wheel_config_path,
+            " controller_config_file:=",
+            controller_config_path,
+            " battery_config_file:=",
+            battery_config_path,
+        ]
+    )
+    robot_description = {"robot_description": robot_description_content}
+
+    control_node = Node(
+        package="controller_manager",
+        executable="ros2_control_node",
+        parameters=[robot_description, controller_config_path],
+        remappings=[
+            ("panther_base_controller/cmd_vel_unstamped", "/cmd_vel"),
+        ],
+        condition=UnlessCondition(use_sim),
+    )
+
+    robot_state_pub_node = Node(
+        package="robot_state_publisher",
+        executable="robot_state_publisher",
+        output="both",
+        parameters=[robot_description],
+    )
+
+    robot_controller_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=[
+            "panther_base_controller",
+            "--controller-manager",
+            "/controller_manager",
+            "--controller-manager-timeout",
+            "120",
+        ],
+    )
+
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=[
+            "joint_state_broadcaster",
+            "--controller-manager",
+            "/controller_manager",
+            "--controller-manager-timeout",
+            "120",
+        ],
+    )
+
+    # Delay start of robot_controller after joint_state_broadcaster
+    delay_robot_controller_spawner_after_joint_state_broadcaster_spawner = RegisterEventHandler(
+        event_handler=OnProcessExit(
+            target_action=joint_state_broadcaster_spawner,
+            on_exit=[robot_controller_spawner],
+        )
+    )
+
+    imu_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=[
+            "imu_broadcaster",
+            "--controller-manager",
+            "/controller_manager",
+            "--controller-manager-timeout",
+            "120",
+        ],
+    )
+
+    # Delay start of imu_broadcaster after robot_controller
+    # when spawning without delay ros2_control_node sometimes crashed
+    delay_imu_broadcaster_spawner_after_robot_controller_spawner = RegisterEventHandler(
+        event_handler=OnProcessExit(
+            target_action=robot_controller_spawner,
+            on_exit=[imu_broadcaster_spawner],
+        )
+    )
+
+    actions = [
+        declare_wheel_config_path_arg,
+        declare_controller_config_path_arg,
+        declare_battery_config_path_arg,
+        declare_use_sim_arg,
+        declare_simulation_engine_arg,
+        SetParameter(name="use_sim_time", value=use_sim),
+        control_node,
+        robot_state_pub_node,
```

Comment:
```suggestion
        robot_state_pub_node,
```
I guess we should add conditions and a param if we want to launch default robot description like in ROS 1